com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/CopyCustomResourcesBuilder.java

```diff
@@ -30,8 +30,7 @@ public Task create(IResource input) throws CompileExceptionError {
         String[] resources = properties.getStringArrayValue("project", "custom_resources", new String[0]);
 
         TaskBuilder b = Task.newBuilder(this)
-                .setName("Copy Custom Resources")
-                .disableCache();
```

Comment:
Instead of disabling cache for a few resource types we enable it just for a few of them